components/button/button-mixin.js

```diff
@@ -5,37 +5,64 @@ export const ButtonMixin = superclass => class extends FocusVisiblePolyfillMixin
 	static get properties() {
 		return {
 			/**
-			 * Indicate expansion state of a collapsible element
```

Comment:
I ignored a lot of the properties in `ButtonMixin` since they don't need to be in the properties table/weren't currently appearing in the README properties table (the accessibility ones generally appear in the `Accessibility Properties` table as well). An alternative to this is if we went with the exclusion functionality mentioned previously we could leave them with jsdoc comments and still have them excluded from the properties table